### DISCUSSION TITLE
How to deactivate https for dev-environment

### DISCUSSION THREAD

**Discussion by feendrache** [api-platform/api-platform#1855] [Q&A]
Hey there,
i'm currently trying to evaluate if Api Platform is the right thing for a new project.
My problem is: i need to test it on a dev server without https. 

Can anyone here tell me how to set the platform up for a first test so it won't use https?
It's not called locally, so no localhost exception is working and i'm not fit enough to know where this should/could be set

Greetings

**Comment by henryfoster** [comment]
```symfony serve --no-tls``` Starts a dev server with https disabled.
